src/D-FSL-1.0.xml

```diff
@@ -0,0 +1,380 @@
+        </li>
+        <li>
+          <b>(1)</b>
+          <p>Veränderte Versionen des Programms dürfen Sie nur unter den Bedingungen dieser Lizenz verbreiten
+             oder öffentlich zugänglich machen, so dass Dritte das veränderte Programm insgesamt unter
+             dieser Lizenz nutzen können.</p>
+        </li>
+        <li>
+          <b>(2)</b>
+          <p>Wird das Programm oder ein Teil hiervon mit einem anderen Programm kombiniert, gilt auch die
+             Kombination insgesamt als eine veränderte Version des Programms, es sei denn, das andere
+             Programm ist formal und inhaltlich eigenständig. Ein anderes Programm ist dann als
+             eigenständig anzusehen, wenn es die folgenden Voraussetzungen alle erfüllt:</p>
+        </li>
+        <li>
+          <b>1.</b>
+          <p>Der Source Code der kombinierten Programme muss jeweils in eigenen Dateien vorhanden sein, die
+             keine Bestandteile des anderen Teils enthalten, die über die zur Programmkombination üblichen
+             und erforderlichen Informationen über den anderen Teil hinausgehen, wobei der Source Code des
+             anderen Programms nicht mitgeliefert werden muss.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Der mit dem Programm kombinierte Teil muss auch dann sinnvoll nutzbar sein, wenn er nicht mit dem
+             Programm kombiniert wird, und zwar entweder alleine oder mit sonstigen Programmen. Was als
+             "sinnvoll nutzbar" anzusehen ist, richtet sich nach der Auffassung der betroffenen Fachkreise.
+             Zu den betroffenen Fachkreisen gehören alle Personen, die das Programm oder Programme mit
+             vergleichbarer Funktionalität entwickeln, benutzen, verbreiten oder öffentlich zugänglich
+             machen.</p>
+        </li>
+        <li>
+          <b>(3)</b>
+          <p>Wenn Sie das Programm oder einen Teil hiervon - verändert oder unverändert - zusammen mit einem
+             anderen Programm verbreiten oder öffentlich zugänglich machen, das unter der GNU General
+             Public License (GPL) lizenziert wird, darf das Programm auch unter den Bedingungen der GPL
+             genutzt werden, sofern es mit dem anderen Programm ein "derivative work" im Sinne der GPL
+             bildet. Dabei sollen die Hinweise auf diese Lizenz entfernt und durch einen Hinweis auf die
+             GPL ersetzt werden. Ob bei der Zusammenstellung ein "derivate work" im Sinne der GPL entsteht,
+             beurteilt sich nach Ziffer 2 b) der GPL. Diese Bestimmung lautet: "You must cause any work
+             that you distribute or publish, that in whole or in part contains or is derived from the
+             Program or any part thereof, to be licensed as a whole at no charge to all third parties under
+             the terms of this License." Die GPL kann abgerufen werden unter
+             http://www.fsf.org/licenses/gpl.</p>
```

Comment:
Upstream is inconsistent here, using [no wrapping chars in the HTML][1] and [`<>` in the text][2].   I recommend using `<alt>` or `<optional>` to cover both cases, and have no opinion on which form should be canonical.

[1]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/deutsche-freie-software-lizenz
[2]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/D-FSL-1_0_de.txt/view